lib/src/textArea/textArea.component.scss

```diff
@@ -0,0 +1,71 @@
+@import "~@sebgroup/bootstrap/scss/variables";
+
```

Comment:
you don't need 80% of this css. just use seb bootstrap classes correctly.

Reply:
I remember we discussed the fact that we are moving to bootsrap specific classes, but this current components are a reflection of what we currently have in react component. I don't we should proceed with those changes in these new component because it will introduce contradiction. We will upgrade and introduce the refactoring as soon as we refactored the react one and vice-versa.